app/src/main/java/com/rokt/roktdemo/ui/layouts/model/DemoModeConfig.kt

```diff
@@ -0,0 +1,17 @@
+package com.rokt.roktdemo.ui.layouts.model
+
+data class DemoModeConfig(
+    val layouts: List<DemoLayoutConfig>,
+)
+
+data class DemoLayoutConfig(
+    val layoutId: String,
+    val versionId: String,
+    val targetElementSelector: String,
```

Comment:
I assume we wouldn't need to send targetElementSelector for mobile demo?

Reply:
Currently it is hardcoded, but will keep it for future implementation.